metadata-db-app/src/cmr/metadata_db/data/oracle/granule_table.clj

```diff
@@ -4,13 +4,7 @@
    [clojure.java.jdbc :as jdbc]
    [cmr.metadata-db.data.util :as util]))
 
-(defmulti granule-column-sql
-  "Returns the sql to define provider granule columns"
-  (fn [provider]
-    (:small provider)))
-
-(defmethod granule-column-sql false
-  [provider]
+(def granule-column-sql-false
```

Comment:
can this be changed to 'granule-column-sql-common' since it gets used for both small and non-small provider

Reply:
Yes, the code has been updated.

Reply:
The function is now called create-granule-column-sql

Reply:
I refactored the main function and that function no longer exists.